server_addon/maya/client/ayon_maya/api/menu.py

```diff
@@ -75,13 +75,17 @@ def add_menu():
 
         cmds.setParent("..", menu=True)
 
-        if project_settings["core"]["tools"]["ayon_menu"].get(
-            "version_up_current_workfile"):
-                cmds.menuItem(divider=True)
-                cmds.menuItem(
-                    "Version Up Workfile",
-                    command=lambda *args: version_up_current_workfile()
-                )
+        try:
+            if project_settings["core"]["tools"]["ayon_menu"].get(
```

Comment:
well, you could even ditch this if with the try/except :)

Reply:
But `version_up_workfile` is a toggled button in the core setting.  Users can choose to turn on/off in the setting.

Reply:
Ah yeah, sorry. I was over-optimizing :)